src/hooks/useMenuInitialFocus.ts

```diff
@@ -25,7 +32,11 @@ export const useMenuInitialFocus = (open: boolean, onOpen?: Callback, providedRe
     if (!openingKey || !containerRef.current) return
 
     const iterable = iterateFocusableElements(containerRef.current)
-    if (['ArrowDown', 'Space', 'Enter'].includes(openingKey)) {
+
+    if (openingKey === 'mouse-click') {
```

Comment:
If the menu is opened with a click, we should not focus any items. The focus should remain on the anchor for natural focus management. For example: If you press Tab, it would move to the next element after the Menu instead of entering the Menu